digits/config/store_option.py

```diff
@@ -15,19 +18,34 @@ def prompt_title(cls):
         return 'Model Store URL list'
 
     @classmethod
+    def prompt_message(cls):
+        return 'What Model Stores should DIGITS look for models?'
+
+    @classmethod
     def visibility(cls):
-        return config_option.Visibility.HIDDEN
+        return config_option.Visibility.DEFAULT
+
+    @classmethod
+    def validate(cls, value):
+        valid_url_list = list()
+        if isinstance(value, str):
+            url_list = value.split(',')
+            for url in url_list:
+                if url is not None and urlparse(url).scheme != "" and not os.path.exists(url):
+                    valid_url_list.append(url)
```

Comment:
The validate method is now throwing BadValue exception whenever it encounters invalid url inside url_list loop.